### PR TITLE
Feature/pool 870 Implemented a simpler Comptroller

### PR DESCRIPTION
**Pull request by asselstine**

### CODE REVIEW

File: contracts/comptroller/ComptrollerV2ProxyFactory.sol [hunk 1]
```diff
@@ -0,0 +1,45 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.6.0 <0.7.0;
+
+import "./ComptrollerV2.sol";
+import "../external/openzeppelin/ProxyFactory.sol";
+
+/// @title Stake Prize Pool Proxy Factory
+/// @notice Minimal proxy pattern for creating new yVault Prize Pools
```
Comment (aodhgan) [inline contracts/comptroller/ComptrollerV2ProxyFactory.sol:9]: yVault - stale comment?

File: contracts/comptroller/ComptrollerV2.sol [hunk 2]
```diff
@@ -0,0 +1,207 @@
+// SPDX-License-Identifier: GPL-3.0
+
+pragma solidity >=0.6.0 <0.7.0;
+
+import "@openzeppelin/contracts-upgradeable/math/SafeMathUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/utils/SafeCastUpgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC20/IERC20Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/proxy/Initializable.sol";
+import "@pooltogether/fixed-point/contracts/FixedPoint.sol";
+import "@nomiclabs/buidler/console.sol";
```
Comment (aodhgan) [inline contracts/comptroller/ComptrollerV2.sol:10]: Unused import?

Comment (asselstine) [reply contracts/comptroller/ComptrollerV2.sol:10]: Yes it is!  ✂️

File: contracts/comptroller/ComptrollerV2ProxyFactory.sol (hunk 1 above)
Comment (asselstine) [reply contracts/comptroller/ComptrollerV2ProxyFactory.sol:9]: Looks like it

**Pull request merged**